docs/mdbook/src/usage/create-table.md

```diff
@@ -0,0 +1,43 @@
+# DataFusion Create Table
+
+It's easy to create DataFusion tables from a variety of data sources.
+
+## Create table from Python Dictionary
```

Comment:
Do we want to capitalize "table" here and the following sections?

Reply:
Good catch, updated throughout.